dashboard/app/models/plc/course_unit.rb

```diff
@@ -68,4 +68,14 @@ def determine_preferred_learning_modules(user)
 
     default_module_assignments
   end
+
+  def launch
```

Comment:
Add a small comment with some context on how this will be used and what it does (e.g. run manually in the rails console, to have a new course show up to enrollees)